monoidal-containers.cabal

```diff
@@ -54,12 +54,13 @@ library
                        hashable >= 1.2 && < 1.4,
                        lens >=4.4 && <5,
                        newtype >=0.2 && <0.3,
-                       semigroups >= 0.18 && < 0.20
+                       semigroups >= 0.18 && < 0.20,
+                       semigroupoids >= 5.0 && < 5.4
```

Comment:
I don't recall why I picked `>= 5.0`. `lens 4.4` requires `semigroupoids ==4.*` so maybe `>= 4.0` is better?
AFAICT Map/IntMap instances were always present, and I added CPP for the HashMap ones .